src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginEntry/FileOriginEntryViewModel.cs

```diff
@@ -83,5 +98,23 @@ public class FileOriginEntryViewModel : AViewModel<IFileOriginEntryViewModel>, I
             .Merge(interval.Select(_ => LastInstalledDate))
             .Select(date => date.Equals(DateTime.MinValue) ? "-" : date.Humanize())
             .ToProperty(this, vm => vm.DisplayLastInstalledDate, scheduler: RxApp.MainThreadScheduler);
+
+        ViewModCommand = ReactiveCommand.Create<NavigationInformation>(info =>
+        {
+            // Note(sewer): Design currently doesn't require we scroll to item,
+            //              (it's challenging) so just navigating to correct
+            //              view is enough.
+            var pageData = new PageData()
+            {
+                FactoryId = LoadoutGridPageFactory.StaticId,
+                Context = new LoadoutGridContext()
+                {
+                    LoadoutId = loadoutId,
+                }
+            };
+
+            var behavior = workspaceController.GetOpenPageBehavior(pageData, info, Optional<PageIdBundle>.None);
```

Comment:
You should pass in the `IdBundle` of the parent FileOriginsPage

Reply:
Can we have some docs somewhere on what this provides us? I think it's for identifying the current view so we replace the right tab, panel etc. I only figured from reading source, it's not documented anywhere.

Anyway sorted.